javascript/ql/lib/semmle/javascript/frameworks/TrustedTypes.qll

```diff
@@ -0,0 +1,68 @@
+/** Module for working with uses of the Trusted Types API. */
```

Comment:
```suggestion
/** Module for working with uses of the [Trusted Types API](https://w3c.github.io/webappsec-trusted-types/dist/spec/). */
```

Reply:
Added a link to the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Trusted_Types_API) instead as spec isn't that useful for our purposes.